meetups/brno.yml

```diff
@@ -1,29 +1,51 @@
-- name: Začátečnický kurz <br> Podzimní pondělí 2017
-  registration:
-    url: https://docs.google.com/forms/d/1ijtXfsv2nIj0zeCWGKcRlHvZcWc_rvktktITdg9GZJs
-    end: 2017-09-03
-  start: 2017-09-25
-  end: 2017-12-18
+# Be careful, new meetups should be added to the end of this file
```

Comment:
Dokumentace o věcech, které mění organizátorky, je zatím všude česky – viz README. (Tam by to taky stálo za to doplnit.)

Reply:
Opraveno a upraveno i v README